### ISSUE TITLE
Todo Explorer: Ignores '@' (at) sign as a marker

### ISSUE THREAD

**Issue by filcuk** [rubberduck-vba/Rubberduck#5251]
**Rubberduck version information**
Version 2.4.1.4627
OS: Microsoft Windows NT 10.0.18362.0, x64
Host Product: Microsoft Office x86
Host Version: 16.0.11328.20438
Host Executable: MSACCESS.EXE

**Description**
ToDo ignores '@' marker

**Screenshots**
![image](https://user-images.githubusercontent.com/1339532/67777839-3c77de00-fa5a-11e9-8387-0a95a09dcd64.png)

**Logfile**
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/3784102/RubberduckLog.txt)

**Comment by retailcoder** [comment]
Indeed, the @ isn't part of the comment, but then we could probably grab the parent node and find markers there and it would just work I think.
